rides/utils.py

```diff
@@ -27,11 +32,14 @@ def wrapped_function(*args, **kwargs):
         last = str(session["userinfo"].get("family_name", ""))
         first = str(session["userinfo"].get("given_name", ""))
         picture = str(session["userinfo"].get("picture", ""))
+        email = str(session["userinfo"].get("email",""))
         auth_dict = {
             "first": first,
             "last": last,
             "uid": uid,
-            "picture": picture
+            "picture": picture,
+            "slack" : 'N/A',
```

Comment:
Why can't this just be `None`?

Reply:
I'm pretty sure the postgresql database is set to not take None/Null, and no matter what it still throws the value into the Slack API to see if it works, and idk if putting None into it will do something bad

Reply:
https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/c03240516eeb3dacaf7ff8e3f1deb1ceaba85e31